abs.yaml

```diff
@@ -0,0 +1,2 @@
+aggregate_check: false
```

Comment:
is it needed ?

Reply:
Charles works this way. This allows nbot having to create the submodule in aggregate since it's ompletely useless anyway for the prefect workflow to run.